Library/Homebrew/cmd/cleanup.rb

```diff
@@ -19,14 +19,9 @@ def cleanup
 
   def cleanup_logs
     return unless HOMEBREW_LOGS.directory?
-    prune = ARGV.value "prune"
-    if prune
-      time = Time.now - 60 * 60 * 24 * prune.to_i
-    else
-      time = Time.now - 60 * 60 * 24 * 7 * 2 # two weeks
-    end
+    prune = prune_flag 14 # two weeks unless overridden by argument
```

Comment:
So it should be possible to handle the logic inside the `prune?` method I believe?